### PR TITLE
add security policy

### PR DESCRIPTION
**Pull request by VigiZhang**
### PR types
Others

### PR changes
Docs

### Describe
Including SECURITY policy and advisory.

### CODE REVIEW

File: SECURITY_cn.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# 安全使用飞桨
+
+
+
+本文将对飞桨模型及代码安全进行介绍，并介绍如何向飞桨提报漏洞。
+
+## 飞桨模型安全
+
+##
```
Comment (luotao1) [inline SECURITY_cn.md]: 第9行多余

File: SECURITY_cn.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### 安全问题认定说明
+
+飞桨在计算图的过程中，由于模型可以执行任何计算，操作文件，进行网络通信等功能，可能造成内存耗尽，死锁等情况发生，这将导致飞桨产生一些非预期的行为。我们认为只有当这些行为超出了所涉及的操作意图时才算作是安全问题。
+
+飞桨框架代码中对于一些非预期的参数和行为会进行检查，Python代码中以抛出异常为形式，C++代码中以返回错误状态为形式。这些情况下，飞桨代码的退出是干净的，但仍可能会因此造成拒绝服务，然而由于飞桨的处理是预期且正确的，所以造成这些情况并不算作是安全问题。
+
+如果输入非预期的参数后，对飞桨代码造成了内存破坏，或者非干净退出，这类行为被认定为存在安全问题。
+
+### [安全公告](https://github.com/PaddlePaddle/Paddle/security/advisories.md)
```
Comment (luotao1) [inline SECURITY_cn.md]: 中文使用链接：https://github.com/PaddlePaddle/Paddle/security/advisories.md
英文使用链接：
https://github.com/PaddlePaddle/Paddle/security/README.md
两者对应么？

**Review by luotao1 — COMMENTED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**